src/Sulu/Bundle/ContentBundle/Resources/public/js/components/internal-links/main.js

```diff
@@ -232,6 +232,30 @@ define([], function() {
             return [this.options.url, delimiter, this.options.idsParameter, '=', (data || []).join(',')].join('');
         },
 
+        overlayOkCallback: function () {
+            this.sandbox.emit('husky.column-navigation.' + this.options.instanceName + '.get-marked', function (markedCollections) {
+                var data = this.sandbox.util.deepCopy(this.getData()),
+                    markedCollectionIds = markedCollections.map(function(markedCollection) {
+                        return markedCollection.id;
+                    });
+
+                markedCollections.forEach(function(markedCollection) {
+                    if ($.inArray(markedCollection.id, data) < 0) {
+                        selectLink.call(this, markedCollection);
+                    }
+                }.bind(this));
+
+                data.forEach(function (item) {
+                    if ($.inArray(item, markedCollectionIds) < 0) {
+                        this.removeHandler(item);
+                    }
+                }.bind(this));
+
+                this.sandbox.stop(getId.call(this, 'columnNavigation'));
```

Comment:
Is there an element with the id `columnNavigation`? Does this line actually do anything? Is it required to stop the column navigation this way? Isn't the overlay already stopping all sub components?